dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -5207,9 +5208,58 @@ int processFrom(bool& isUnion, SELECT_LEX& select_lex, gp_walk_info& gwi, SCSEP&
 
         // trigger system catalog cache
         if (columnStore)
+        {
           gwi.csc->columnRIDs(
               make_table(table_ptr->db.str, table_ptr->table_name.str, lower_case_table_names), true);
-
+        }
+        else
+        {
+          for (uint j = 0; j < table_ptr->table->s->keys; j++)
+          {
+            // for (uint i = 0; i < table_ptr->table->s->key_info[j].usable_key_parts; i++)
```

Comment:
nit: Is this commented code needed for further develop? If not, it's probably better to delete it, imo...

Reply:
Removed